server.js

```diff
@@ -3082,6 +3083,52 @@ cache(function(data, match, sendBadge, request) {
   })}
 ));
 
+// StackExchange integration.
+camp.route(/^\/stackexchange\/([^\/]+)\/([^\/])\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var site = match[1]; // eg, stackoverflow
+  var info = match[2]; // either `r`
+  var item = match[3]; // eg, 232250
+  var format = match[4];
+  var path;
+  if (info === 'r') {
+    path = 'users/'+item;
+  } else if (info === 't') {
+    path = 'tags/'+item+'/info';
+  }
+  var options = {
+    method: 'GET',
+    uri: 'https://api.stackexchange.com/2.2/'+path+'?site='+site,
+    encoding: null
```

Comment:
Replace this line with `gzip: true`.

Change the required version of request from `"~2.34.0"` to `"~2.54.0"` in `package.json`. Do `npm install request`.